.storybook/viewport.js

```diff
@@ -7,62 +7,91 @@ export default {
     styles: {
       width: '320px',
       height: '480px'
-    },
-    type: 'mobile'
+      },
+      type: 'mobile'
   },
-  iPhoneSE: {
+  iPhoneSE: { 
     name: 'Apple iPhone SE',
     styles: {
       width: '320px',
-      height: '568px'
+      height: '568px',
+      borderTopWidth: '95px',
+      borderBottomWidth: '95px',
+      borderLeftWidth: '12.5px',
+      borderRightWidth: '12.5px',
+      borderRadius: '55px'     
     },
     type: 'mobile'
   },
   Note9: {
     name: 'Samsung Galaxy Note 9',
     styles: {
       width: '360px',
-      height: '740px'
+      height: '740px',
+      borderTopWidth: '35px',
+      borderBottomWidth: '25px',
+      borderLeftWidth: '7.5px',
+      borderRightWidth: '7.5px',
+      borderRadius: '25px'
     },
     type: 'mobile'
   },
   iPhone6: {
     name: 'Apple iPhone 6',
     styles: {
       width: '375px',
-      height: '667px'
+      height: '667px',
+      borderTopWidth: '95px',
+      borderBottomWidth: '95px',
+      borderLeftWidth: '12.5px',
+      borderRightWidth: '12.5px',
+      borderRadius: '55px'     
     },
     type: 'mobile'
   },
   iPhone11Pro: {
     name: 'Apple iPhone 11 Pro',
     styles: {
-      width: '375px',
-      height: '812px'
+      width: '414px',
+      height: '896px',
```

Comment:
Can you cite sources for this change?

Reply:
The IPhone 11 had the same dimensions as compared  to the IPhone 11 Pro before I made these changes. In reality, IPhone 11 Pro is just slightly smaller than IPhone 11. (5.8 v/s 6.1). Will make the necessary changes!

Reply:
Yeah, screen sizes are generally independent of screen resolutions. Changing the resolution to mimic physical dimensions is not correct.

Reply:
Agreed. Will revert back this change.

Reply:
@dhruvkb Reverted. Fixed!